### PR TITLE
apt-dater 1.0.3

### PR DESCRIPTION
**Pull request by MikeLund**
All the checkmarks should be good. But please read https://github.com/Homebrew/homebrew-core/issues/5762 before accepting. If everything is fine in regards to that link, then it should be fine to accept. Otherwise putting this on hold. Thanks!

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/apt-dater.rb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 class AptDater < Formula
   desc "Manage package updates on remote hosts using SSH"
   homepage "https://www.ibh.de/apt-dater/"
-  url "https://downloads.sourceforge.net/project/apt-dater/apt-dater/0.9.0/apt-dater-0.9.0.tar.gz"
-  sha256 "10b8335c156dabae249aa071cf8689900fae325c2e9540e36a840a2a77d3eeb4"
+  url "https://github.com/DE-IBH/apt-dater.git",
+      :tag => "v1.0.3",
+      :revision => "fdad3b43e50bc3582e391d6feabae150e133b3b3"
```
Comment (MikeMcQuaid) [inline Formula/apt-dater.rb]: Can this build from a archive tarball rather than a Git repository?

Comment (MikeLund) [inline Formula/apt-dater.rb]: Maybe can use https://github.com/DE-IBH/apt-dater/releases, but why though? I thought git repo is better for Homebrew.

**Comment by MikeLund** [comment]
Actually is probably best regardless to close without merging.
The config file syntax has completely changed, and it's quite confusing with the numerous-config-files thing. So maybe not very user-friendly to merge :/

**Comment by MikeMcQuaid** [comment]
@MikeLund That's fine, we'd rather have the newest version.

**Comment by MikeLund** [comment]
Sorry for time waste but I regret this and think it should not continue; current version is a bit too buggy, old version was fine and stable, this adds nothing. The developer also seems to be away, and his homepage doesn't even mention program anymore, so it's a bit dead in the water.
Will retry once everything is going good for them again.

**Comment by MikeMcQuaid** [comment]
No worries, thanks for letting us know.

**Pull request closed without merging**